packages/task/src/node/test-resources/task-long-running.bat

```diff
@@ -0,0 +1,10 @@
+@echo off
+
+@echo off
+for /l %%x in (1,1,300) do (
+   echo tasking... %%x
+   @REM sleep for ~1s
+   @REM  see: https://stackoverflow.com/questions/735285/how-to-wait-in-a-batch-script
+   ping 192.0.2.2 -n 1 -w 1000> nul
```

Comment:
weird IP why not 127.0.0.1 ?

Reply:
The answer is in the link above: basically this is garanteed, by definition, to be an address that can't be assigned. So we are certain that no one will answer and that we will spend the "1000" ms waiting. So basically a poor man's "sleep". If we used 127.0.0.1, it would answer immediately and not wait for 1000ms.

Reply:
right OK!